modules/lazyload/module.php

```diff
@@ -54,6 +54,13 @@ private function update_element_attributes( Element_Base $element ) {
 
 		if ( $keys ) {
 			$background_image_url = Utils::get_array_value_by_keys( $settings, $keys );
+			if ( ! $background_image_url ) {
```

Comment:
Why do you have a separated condition when the $background_image_url does not exist?
why not if/else?